coreplugins/cesiumion/public/components/IonAssetLabel.jsx

```diff
@@ -0,0 +1,12 @@
+import React, { PureComponent, Fragment } from "react";
+import { AssetStyles } from "../defaults";
+
+const IonAssetLabel = ({ asset, showIcon = false, ...options }) => (
+	<Fragment>
+		{showIcon && <i className={`${AssetStyles[asset].icon}`} />}
+		{"  "}
```

Comment:
Can we add some margin if we need space between elements instead of using space characters?